src/Compilers/CSharp/Portable/Parser/Lexer.cs

```diff
@@ -2237,7 +2240,7 @@ private void LexSyntaxTrivia(bool afterFirstToken, bool isTrailing, ref SyntaxLi
 
                             // normal single line comment
                             this.ScanToEndOfLine();
-                            var text = TextWindow.GetText(false);
+                            var text = TextWindow.GetText(poolTrivia);
```

Comment:
This makes sense. I'll add the safeguard.

https://github.com/mono/monodevelop/blob/master/main/src/core/MonoDevelop.Startup/MonoDevelop.Startup/MonoDevelopMain.cs#L1-L27

This is a sample of a copyright header that reproduces the scenario this commit is trying to fix. In this case, the comments are pretty short and the only thing that could change is the author, which has a high chance of re-appearing.

Nevertheless, for the general case, it would be great to check the comment size.